blocked-edges/4.3.1.yaml

```diff
@@ -0,0 +1,4 @@
+to: 4.3.1
+from: 4\.2\..*
+# 4.2 -> 4.3 updates occasionally hit RequiredPoolsFailed, fixed in 4.2.18 and rc.0, but not in 4.2.16: https://bugzilla.redhat.com/show_bug.cgi?id=1782152 https://bugzilla.redhat.com/show_bug.cgi?id=1782149
```

Comment:
This doesn't apply to 4.3.1, since the 4.2 source is 4.2.18:

```console
$ oc adm release info quay.io/openshift-release-dev/ocp-release:4.3.1-x86_64 | grep Upgrade
  Upgrades: 4.2.18, 4.3.0-rc.0, 4.3.0-rc.3, 4.3.0
```

More on that in c641bbdc4 (#60).